main/browser-config.ts

```diff
@@ -69,9 +69,10 @@ export default class BrowserConfig {
         if (this.window_state === null) {
             return null;
         }
-        win.once('closed', () => {
+        win.on('resize', () => {
```

Comment:
Hmm... Document says `'close'` event

> Emitted when the window is going to be closed. It's emitted before the beforeunload and unload event of the DOM. 

unlike `'closed'` event.  So, it may be a bug of Electron if 'Object has been destroyed' error occurs before window is unloaded.  OK, I'll introduce `'resize'` event as you said.  Thank you for explanation :smile: